src/app/services/baw-api/baw-api.service.ts

```diff
@@ -71,113 +104,98 @@ export class BawApiService<Model extends AbstractModel> {
   }
 
   public constructor(
-    protected http: HttpClient,
     @Inject(API_ROOT) protected apiRoot: string,
-    @Inject(STUB_MODEL_BUILDER)
-    classBuilder: AbstractModelConstructor<Model>,
+    @Inject(IS_SERVER_PLATFORM) protected isServer: boolean,
```

Comment:
if i've understood this right, there are no child classes any more? so why `protected`?

Reply:
`BawFormApiService` extends `BawApiService`. Keeping this `protected` may be useful in the future

Reply:
Although potentially it should import `BawApiService` instead to be more inline with everything else

Reply:
Form service seems like an actual sub-class of enhanced functionality. Keep as is